src/Utils/getSignalLabel.ts

```diff
@@ -2,9 +2,24 @@ export interface CollectorSignals {
   primaryLabel?: string | null | undefined
 }
 
-export const getSignalLabel = (collectorSignals: CollectorSignals) => {
+interface SignalLabelProps {
+  collectorSignals: CollectorSignals
+  hideSignals?: boolean
+}
+
+export const getSignalLabel = ({
+  collectorSignals,
+  hideSignals,
+}: SignalLabelProps) => {
   const { primaryLabel } = collectorSignals
 
+  if (
+    hideSignals &&
+    (primaryLabel === "CURATORS_PICK" || primaryLabel === "INCREASED_INTEREST")
+  ) {
+    return ""
+  }
```

Comment:
I don't fully understand why we have this single hideSignals value for multiple kinds of signals that could be hidden. Like we might want to suppress curators pick on a curators pick page, but still show increased interest, right? Would it make sense to just make this `hideSignals` specific to the kinds of signals we are hiding?